source/code/media_types.md

```diff
@@ -71,21 +58,23 @@ The link can be either text, or an image (icon):
 
 ```html
 <!-- Worded link -->
-<a href="../videos/everyday-theology-043.xhtml#video-043">Video on Adoption</a>
+<a href="../videos/everyday-theology-045.xhtml#video-045">Video on Adoption</a>
 
 <!-- Icon link -->
-<a href="../videos/everyday-theology-043.xhtml#video-043"><img alt="Play Video" src="../images/video.png" /></a>
+<a href="../videos/everyday-theology-045.xhtml#video-045"><img alt="Play Video" src="../images/video.png" /></a>
 ```
 
 This is our standard video icon: ![video.png icon](/assets/images/uploads/video.png) Right-click and "Save Image As..." to download and use it in your book.
 
-For Linked videos, the video markup must be placed in individual XHTML files (one per video) in a **videos** folder.
+For Linked videos, the video markup must be placed in individual XHTML files (one per video) in the **OEBPS/videos/** folder.
 
-![OEBPS/videos folder](/assets/images/uploads/videos-folder.png)
+![XHTML files in OEBPS/videos folder](/assets/images/uploads/videos-folder.png)
 
 For consistency, the XHTML file name should be the same as the MP4 and WEBM file names (which, in turn, should be the same as the SVG file name).
 
-The only difference in the markup from [Embedded Videos](#Embedded-Videos) is the addition of an `id` attribute to the `video` element for linking to from the text. Each `id` value must be unique.
+The differences in the markup from [Embedded Videos](#Embedded-Videos) are:
+- the addition of an `id` attribute to the `video` element for linking to from the text. (Each `id` value must be unique.)
```

Comment:
Small suggestion: "for linking to and from the text"

Reply:
I changed the bullet point to "the addition of an id attribute to the video element (to which the link's href should point.)"

I thought the admonition to keep the `id`s unique was unnecessary.

And looking at it more right now, the parenthetical specification that the `href` should point to the `id` is also probably unnecessary. I mean, it goes without saying, right?

I didn't go with your wording since there isn't a backlink from the video to the link.

What do you think?

Reply:
That sounds good, I just reread what was originally written and understand now... I was reading it wrong. I made this way more complicated than it needed to be. :)

Reply:
No, you didn't make it more complicated, I appreciate your input!